### PR TITLE
chore: Add lerna as a peer dependency

### PR DESCRIPTION
**Pull request by idandagan1**
According to the issue #18 - I've added lerna as peer dependency

### CODE REVIEW

**Comment by 0livare** [comment]
Hi @idandagan1, thanks for the PR!

File: package.json [hunk 1]
```diff
@@ -35,5 +35,8 @@
     "execa": "^2.0.3",
     "node-emoji": "^1.10.0",
     "ora": "^3.4.0"
+  },
+  "peerDependencies": {
+    "lerna": "^3.22.1"
```
Comment (0livare) [inline package.json]: I believe this version should be `3.x`.  We want to support everything except a major version of lerna that could change the API.

**Review by 0livare — CHANGES_REQUESTED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -2,11 +2,994 @@
 # yarn lockfile v1
 
 
+"@babel/code-frame@^7.0.0":
```
Comment (0livare) [inline yarn.lock:5]: I also don't understand why all these changes were made to the lock file.  Shouldn't a peer dependency not affect this lock file at all?

Is there a chance you `yarn add`ed it as a `devDependency` and then changed the `package.json` for it to be a `peerDependency`?

Comment (idandagan1) [reply yarn.lock:5]: peer dependencies only ensure that lerna should be the exact same ones with the ones used by your parent package in the dependency tree.

Anyway, I ran the command 
```
yarn add lerna@3.x -P
```
I've double-checked that I didn't install it by mistake as "dev" - so I've cloned the repo again, ran this command and got the same result.

**Pull request merged**